src/controllers/userController.js

```diff
@@ -0,0 +1,90 @@
+const userService = require('../services/userService');
+
+const get = (req, res) => {
+  const users = userService.getAll();
+
+  if (!users) {
+    res.send(users);
```

Comment:
This condition seems to be incorrect. You're checking if there are no users, and if so, you're sending an empty list (which is a truthy value) and then trying to send a 404 status code. However, once a response is sent, you cannot send another one. It will cause an error. You should only send the 404 status code if there are no users.